packages/valory/skills/abstract_round_abci/tests/test_models.py

```diff
@@ -490,14 +490,8 @@ def test_synchronized_data_db(self, *_: Any) -> None:
                 "all_participants": "0x0",
             }
             shared_state.setup()
-            assert (
-                shared_state.synchronized_data.db.get_strict("safe_contract_address")
-                == "0xsafe"
-            )
-            assert (
-                shared_state.synchronized_data.db.get_strict("oracle_contract_address")
-                == "0xoracle"
-            )
+            for key, value in mock_params.setup_params.items():
+                assert shared_state.synchronized_data.db.get_strict(key) == value
```

Comment:
Unrelated to the PR, but a check for `all_participants` was missing, so I updated the test to include it.